apps/demo-game/src/pages/play/cockpit.tsx

```diff
@@ -161,124 +160,229 @@ function Cockpit() {
   const data_portfolio = [
     {
       cat: 'Savings',
-      val: `CHF ${resultFacts?.assets.bank.toFixed(2)}`,
+      val: `CHF ${assets.bank.toFixed(2)}`,
     },
     {
       cat: 'Bonds',
-      val: `CHF ${resultFacts?.assets.bonds.toFixed(2)}`,
+      val: `CHF ${assets.bonds.toFixed(2)}`,
     },
     {
       cat: 'Stocks',
-      val: `CHF ${resultFacts?.assets.stocks.toFixed(2)}`,
+      val: `CHF ${assets.stocks.toFixed(2)}`,
     },
     {
       cat: 'Total',
-      val: `CHF ${resultFacts?.assets.totalAssets.toFixed(2)}`,
+      val: `CHF ${assets.totalAssets.toFixed(2)}`,
     },
   ]
 
-  const header = (
-    <div className="rounded border p-4">
-      <div className="font-bold">
-        Playing as {self?.name} in game {currentGame.id}
-      </div>
+  // TODO(JJ):
+  // - Level is not consistent with component
+  // Component wants a number, level is an object, though.
+  // console.log(self)
+  // - Do onclick logic
+  // - PlayerDisplay is not ideal/nice yet
+
+  const tabs = [
```

Comment:
stuff like this I normally put outside the function, as it is constant and does not need to be inside the render loop